README.md

```diff
@@ -1,19 +1,77 @@
 <br />
 <p align="center">
-  This repository contains a fully usable <a href="https://github.com/BookStackApp/BookStack"><b>BookStack</b></a> with <a href="https://github.com/traefik/traefik"><b>Traefik</b></a> as a reverse proxy. Additionally, Traefik uses <a href="https://letsencrypt.org/"><b>Let's Encrypt</b></a> as an ACME provider to automate certificate generation. The whole stack can be set up with the included <a href="https://github.com/ansible/ansible"><b>Ansible</b></a> code.
+  This repository contains a fully usable <a href="https://github.com/BookStackApp/BookStack"><b>BookStack</b></a> with <a href="https://github.com/traefik/traefik"><b>Traefik</b></a> as a reverse proxy. Additionally, Traefik uses <a href="https://letsencrypt.org/"><b>Let's Encrypt</b></a> as an ACME provider to automate certificate generation.
 </p>
 <br />
 
 ## ⚡️ Installation
 
 1. Make sure you've installed Docker including `docker-compose` support on your target system.
+   You can install `docker-compose` following this instruction: https://docs.docker.com/compose/install/
+
+<br>
 
 2. Clone this repository:
 
    ```bash
-   git clone https://github.com/symnixhq/traefik-bookstack.git
+   sudo git clone https://github.com/symnixhq/traefik-bookstack.git
+   ```
+
+<br>
+
+3. Open the file called `traefik-letsencrypt-bookstack-docker-compose.yml`
+
+<br>
+
+4. Adjust the variables for your configuration. Edit the following values:
+   1. Passwords (declared as "MODIFY_ME")
+   2. E-Mail address (declared as "example.mail@mail.com")
+   3. Domain (declared as "example.url.com")
+   4. Encrypted basic auth string for your Traefik dashboard (declared as "admin:$$2y$$10$$tzGrxnBWcBBCKpFWdvzJDOt/Y4ihpuxXuE9r7mP/eo1MRBGcmV9q2")
```

Comment:
How does the user create their encrypted basic auth string. Should that described here? What's the current value of this string? Wouldn't it be safer to put no value at all in. That way the lazy user cannot just use the password and opens up a security hole, but is forced to create their own password?

Reply:
I've added a documentation part on how you can encrypt a password to use it in your basic auth.